docs/jitserver.md

```diff
@@ -0,0 +1,88 @@
+<!--
+* Copyright (c) 2017, 2020 IBM Corp. and others
+*
+* This program and the accompanying materials are made
+* available under the terms of the Eclipse Public License 2.0
+* which accompanies this distribution and is available at
+* https://www.eclipse.org/legal/epl-2.0/ or the Apache
+* License, Version 2.0 which accompanies this distribution and
+* is available at https://www.apache.org/licenses/LICENSE-2.0.
+*
+* This Source Code may also be made available under the
+* following Secondary Licenses when the conditions for such
+* availability set forth in the Eclipse Public License, v. 2.0
+* are satisfied: GNU General Public License, version 2 with
+* the GNU Classpath Exception [1] and GNU General Public
+* License, version 2 with the OpenJDK Assembly Exception [2].
+*
+* [1] https://www.gnu.org/software/classpath/license.html
+* [2] http://openjdk.java.net/legal/assembly-exception.html
+*
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH
+* Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+
+# JITServer technology (technical preview)
+
+![Start of content that applies to Java 8 and later](cr/java8.png)
+![Start of content that applies to Java 11 and later](cr/java11.png)  
+**Linux&reg; on x86-64 only**
+
+JITServer technology decouples the JIT compiler from the VM and lets the JIT compiler run remotely in its own process. This prevents your Java&trade; application suffering possible negative effects due to CPU and memory consumption caused by JIT compilation.
```

Comment:
When I read this para I got thrown by the "This prevents...". Should we add a qualifier in here? I think we mean that the JITServer technology helps with resources. Maybe say "This capability.." or "This mechanism.."?